packages/web3-providers-http/src/index.ts

```diff
@@ -112,7 +112,7 @@ export default class HttpProvider<
 
 	/* eslint-disable class-methods-use-this */
 	public disconnect() {
-		throw new MethodNotImplementedError();
+		// NO OP
```

Comment:
Why to stop throwign on only this specific method on HttpProvider?
I think we should keep this method as it was, similar to other methods in this class.

Reply:
we should throw error in http instead of no action for disconnect.

Reply:
ok

Reply:
done